### ISSUE TITLE
Recommended firewall settings?

### ISSUE THREAD

**Issue by Kaspre** [NanoTools/easy-nano-node#7]
Thanks for sharing this tool!  Really good starting point for newbies and folks like me that are "a little rusty."

Question:  DO allows you to apply a firewall to your DO droplet, and that seems like it would be a good idea.  But I don't want to block the ports currently in use by Docker/Nano Node/Node Monitor, or anything else that needs to be unblocked.  Do you recommend applying a DO firewall?  What ports should be whitelisted?

Any other security and/or general maintenance tips for keeping our Easy Nano Nodes secure and up-to-date?

**Comment by BitDesert** [comment]
The DO Docker droplet has already UFW enabled. Only the ports 22, 80 and 443 are open.
To allow other nodes to connect to your node you should allow the port 7075 (already done by the easy-nano-node script).
To update your node just execute the setup command again.
If you want notifications when your node stops voting in the network go to http://nanonode.ninja/
I would suggest adding an nginx with Let's Encrypt or Caddy Server for HTTPS.
